core-java-modules/core-java-exceptions/pom.xml

```diff
@@ -0,0 +1,26 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.exception.numberformat</groupId>
+    <artifactId>core-java-exceptions</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>NumberFormatExceptionExamples</name>
```

Comment:
```suggestion
    <name>core-java-exceptions</name>
```

Reply:
As per guidelines, name is optional, but if provided, must be the same as the artifactId (and directory name).